Sources/swift-build/main.swift

```diff
@@ -24,7 +24,11 @@ import Build
 import Get
 
 /// Declare additional conformance for our Options type.
-extension Options: XcodeprojOptions {}
+extension Options: XcodeprojOptions {
+    var xccacheDirectory: String {
+        return Utility.Path.join(path.build, "XcodeProjCache")
+    }
```

Comment:
Even if we do end up using a cache directory for this, I don't think we should use the `.build` directory when doing Xcode project generation, which is otherwise unused.